src/frontend/staff-portal/src/app/shared/components/header/header.component.html

```diff
@@ -8,7 +8,11 @@
            rel="noopener">
           <img src="/assets/gov_bc_logo.jpg"
                alt="Government of BC Logo" />
-        </a>| Ticket Resolution  Management
+        </a>| Ticket Resolution  Management 
+        <span class="badge badge-pill badge-light header"
+            *ngIf="environment !== 'prod'">
+        {{ environment }} v{{ version }}
+      </span>
```

Comment:
display of environment and version in different location than citizen portal, due to overlap with Sign in / Sign out IDIR buttons.  cleared with Business Analyst